src/tests/ftest/scripts/main.sh

```diff
@@ -22,7 +22,8 @@ then
     rm -rf venv
 fi
 
-python3 -m venv venv
+latest_python=$(find /usr/bin -name 'python*' -type f -executable | grep -E '[0-9]$' | sort -r | head -1)
```

Comment:
The other option would be to call `update-alternatives` after installing a later version at https://github.com/daos-stack/daos/blob/master/ci/provisioning/post_provision_config_nodes_EL_8.sh#L23

I think we concluded that my PR installed python3.9 on EL8 but actually it didn't need to so you could also try removing that.